src/www/index.html

```diff
@@ -202,6 +202,14 @@
           <a class="custom-icon" href="https://loklak.org/" target="_blank"><img src="./icons/loklak.png">
           <p class="custom-title">loklak</p></a>
         </div>
+        <div class="custom-menu-item">
+          <a class="custom-icon" href="https://phimp.me/" target="_blank"><img src="./icons/phimp.png">
+          <p class="custom-title">Phimp.me</p></a>
+        </div>
+        <div class="custom-menu-item">
```

Comment:
@arjunpandey2003 Please add a square logo, this logo does not scale properly.

Reply:
sure

Reply:
should i increase the width

Reply:
@arjunpandey2003 Please use a square logo. Do not change width